src/components/common/Card.js

```diff
@@ -0,0 +1,13 @@
+import { Title } from "./Title_Styled";
+
+export function Card({ Image, title, subtitle = "", className, children }) {
+  return (
+    <div className={`${className} card-box`}>
+      {!!Image && Image}
+      <Title as="h5">{title}</Title>
+      {!!subtitle && <Title as="h6">{subtitle}</Title>}
```

Comment:
The value of a variable doesn't need to be a boolean when you need to render a component conditionally, you can use [truthy](https://developer.mozilla.org/en-US/docs/Glossary/Truthy) and [falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy) values:

```suggestion
      {subtitle && <Title as="h6">{subtitle}</Title>}
```

Reply:
Solved